Zend/zend_types.h

```diff
@@ -394,7 +394,8 @@ struct _zend_array {
 #define HT_INVALID_IDX ((uint32_t) -1)
 
 #define HT_MIN_MASK ((uint32_t) -2)
-#define HT_MIN_SIZE 8
+#define HT_MIN_SIZE 2
+#define HT_MIN_SIZE_UNPACKED 8
```

Comment:
``UNPACKED`` is a bad name. We already use ``HASH_MAP`` name it iterators API.
Also the meaning of ``HT_MIN_SIZE`` is not clear now. Should it be renamed into ``HT_MIN_SIZE_PACKED``?